app/controllers/trails_controller.rb

```diff
@@ -7,6 +7,9 @@ def index
   end
 
   def show
+    @lat = Trail.find(params[:id]).latitude
+    @lon = Trail.find(params[:id]).longitude
+    @weather = Weather.weather_info(@lat, @lon)
```

Comment:
@Mcents Do we want latitude and longitude to be a instance variables rather than regular vars? I don't see a `<%= @lat/@long %>` in the views and if we are only passing them through as args for `Weather.weather_info(@lat, @long)` couldn't we just make it `...(lat, long)`? Just for readability